webapp/app/js/filters.js

```diff
@@ -39,11 +39,21 @@ treeherder.filter('stripHtml', function() {
 
 treeherder.filter('linkifyBugs', function() {
     return function(input) {
-        var re = new RegExp('(?:Bug (\\d+))', 'ig');
-        var str = input || '';
-        return str.replace(re,
-            '<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=$1" target="_blank" title="Bug $1">$1</a>'
-        );
+        if (input) {
```

Comment:
If `input` is not undefined, this function will now return undefined, whereas before it would return an empty string. Was this intentional?